week-2/extended-project/banzai/cli_test.rb

```diff
@@ -0,0 +1,106 @@
+require "./cli"
+
+def init
+  puts "init"
+  @task = read_file
+end
+
+def back
+  puts "back"
+  save_file(@task)
+end
+
+RSpec.describe "Test cli" do
+  before(:each) do
```

Comment:
@cdsandoval  please can you explain me how does "before and after" sentences work in these lines?

Reply:
Sure, we call this method before each block of testing examples (it - end) to control the environment in which it is run. Please, let me know if you understand it.